### DISCUSSION TITLE
i3wm on 15inch 3840x2160 display font too small

### DISCUSSION THREAD

**Discussion by tenspd137** [i3/i3#4536] [Q&A]
Hi all -
How do you increase the size of the menus on various programs when running i3?  For example, the 'File' menu on VSCode is now too small, and nothing I change in .Xresources affects that.  I am usung Ubuntu 20.04 / i3wm 4.19.1.  My internet searching has turned up nothing really helpful.  I use i3 on all my other computers, but their screens aren't as pixel dense as this one, so this hasn't been an issue.  Any suggestions?

Thanks!

**Comment by etrigan63** [comment]
Install lxappearance and kvantum-qt to manage the appearance of GTK and QT
apps respectively.
[image: photo]
Carlos Echenique
Photographer at  CE Photographic
P  305.219.2433  <305.219.2433>
E  ***@***.***  ***@***.***>
W  cephotographic.com
<https://mailtrack.io/trace/link/f382bf87caa0955b9821893a598baf320a0c2b7c?url=http%3A%2F%2Fcephotographic.com&userId=3957204&signature=ad6575e3286b82cf>
<https://mailtrack.io/trace/link/180084053a465513b2751d86265ad20a545be8f8?url=http%3A%2F%2Finstagram.com%2Fcephotographic&userId=3957204&signature=41d0d08880a1d0ae>
<https://mailtrack.io/trace/link/5729e59da32a9342cca4b2811ffc7fca036dcb94?url=http%3A%2F%2Ffacebook.com%2Fcephotos&userId=3957204&signature=a9d5792718aacf19>
<https://mailtrack.io/trace/link/c802dce280d0b0d977a57d6e7a9b043e8398aea2?url=http%3A%2F%2Fwww.flickr.com%2Fphotos%2Fechenique&userId=3957204&signature=47d598a765333cc9>

"What after all, is a halo? It's only one more thing to keep clean." -
Christopher Fry.
Create your own email signature
<https://mailtrack.io/trace/link/663e3a91655c16432ed46c18e176ddf4b8139456?url=https%3A%2F%2Fwww.wisestamp.com%2Fcreate-own-signature%2F%3Futm_source%3Dpromotion%26utm_medium%3Dsignature%26utm_campaign%3Dcreate_your_own%26srcid%3D&userId=3957204&signature=329f1be4058ae1de>
‌

**Reply by tenspd137** [reply]
I tried lxappearance.  It doesn't let me actually increase the size of the font in the menu ribbon.  So, for example, in gnome terminal (and many other programs) File, Edit, etc are barely readable.

**Comment by etrigan63** [comment]
Use alacritty instead of gnome-terminal. Configured from a yaml file.

[image: photo]
Carlos Echenique
Photographer at  CE Photographic
P  305.219.2433  <305.219.2433>
E  ***@***.***  ***@***.***>
W  cephotographic.com
<https://mailtrack.io/trace/link/03688f39ab89393b0109c5985bddb4f6eda34394?url=http%3A%2F%2Fcephotographic.com&userId=3957204&signature=f827949352e77164>
<https://mailtrack.io/trace/link/af73ece70e40dff8df5149d2b72b63ebf70df06f?url=http%3A%2F%2Finstagram.com%2Fcephotographic&userId=3957204&signature=5b353e24de482440>
<https://mailtrack.io/trace/link/dfd36da7100b301df5ec37182c0233e74e27c8a0?url=http%3A%2F%2Ffacebook.com%2Fcephotos&userId=3957204&signature=4fb943a8454e3a5a>
<https://mailtrack.io/trace/link/2417fc556fe0c9f946be0bd6485e3e5657a868c6?url=http%3A%2F%2Fwww.flickr.com%2Fphotos%2Fechenique&userId=3957204&signature=a7860102aef429ae>

"What after all, is a halo? It's only one more thing to keep clean." -
Christopher Fry.
Create your own email signature
<https://mailtrack.io/trace/link/3f96886adc586559b47a5f8ebb3e8c056fa112ff?url=https%3A%2F%2Fwww.wisestamp.com%2Fcreate-own-signature%2F%3Futm_source%3Dpromotion%26utm_medium%3Dsignature%26utm_campaign%3Dcreate_your_own%26srcid%3D&userId=3957204&signature=4b0a629755b39da0>
‌

**Comment by michaelmrose** [comment]
This is a good resource for making various applications play nicely with HiDPI

https://wiki.archlinux.org/title/HiDPI Although of course the arch wiki is focused on arch much in the article is generally applicable. For instance attend to this subsection regarding GTK apps.

https://wiki.archlinux.org/title/HiDPI#GDK_3_(GTK_3)

You will almost certainly find the need to tweak  additional things for an optimal experience as you go on.
